python/cuspatial/cuspatial/core/binpreds/contains.py

```diff
@@ -50,6 +47,13 @@ def contains_properly(
         within its corresponding polygon.
     """
 
+    scale = (
+        (test_points_x.std() + test_points_y.std()) / 2.0
+        if len(test_points_x) > 1
+        else 1.0
+    )
```

Comment:
Some comment explaining this?

Reply:
What happens if this heuristic is not good for the provided points? Do we still get correct results?

Reply:
After chatting with @trxcllnt decided to just set this to -1.